packages/participants-svc/src/domain/entities/enums.ts

```diff
@@ -83,7 +83,7 @@ export enum ParticipantChangeTypes {
 	"APPROVE"= "APPROVE",
 	"ACTIVATE"= "ACTIVATE",
 	"DEACTIVATE"= "DEACTIVATE",
-	"ADD_ACCOUNT"= "ADD_ACCOUNT",
+	"CHANGE_ACCOUNT"= "CHANGE_ACCOUNT",
```

Comment:
These should describe the operator action (not just a change), for example:
ADD_ACCOUNT
CHANGE_ACCOUNT_BANK_DETAILS

Reply:
Should be CHANGE_ACCOUNT_BANK_DETAILS instead of CHANGE_ACCOUNT